Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,201 @@
+## Overview
+
+Currently, Oppia E2E tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The GSoC project **Migrate away from Protractor** aims to migrate all the existing E2E tests to WebdriverIO.
+
+This guide is to help contributors in migrating end-to-end tests from Protractor to WebdriverIO during the hybrid state.
+
+## Selectors
+
+A Selector is used to query an element. The WebDriver Protocol provides several selector strategies to query an element. WebdriverIO simplifies them to keep selecting elements simple
```

Comment:
```suggestion
A selector is used to query an element. The WebDriver Protocol provides several selector strategies to query an element. WebdriverIO simplifies them to keep selecting elements simple.
```

Please carefully check your PR for other grammar/capitalization/spelling errors like this one. I've noticed several already

Reply:
Done.